options/locale/locale_en-US.ini

```diff
@@ -61,103 +61,103 @@ cancel = Cancel
 
 [install]
 install = Installation
-title = Initial configuration
-docker_helper = If you are running Gitea inside Docker, please read the <a target="_blank" rel="noopener" href="%s">guidelines</a> carefully before changing anything on this page.
-requite_db_desc = Gitea requires MySQL, MSSQL, PostgreSQL, SQLite3, or TiDB.
+title = Initial Configuration
+docker_helper = If you run Gitea inside Docker, please read the <a target="_blank" rel="noopener" href="%s">documentation</a> before changing any settings.
+requite_db_desc = Gitea requires MySQL, PostgreSQL, MSSQL, SQLite3 or TiDB.
```

Comment:
Why are you inverting the order?

Reply:
That's the actual order of the options in the list.

Reply:
Fair enough.

Reply:
@thehowl can you close or accept this if you agree to keep the review page as clean as possible? Thanks!

Reply:
github won't let me close the conversation in itself, even after approving the pr, so can't do anything about that, sorry :s